tabletools/table_group_summarize.py

```diff
@@ -34,6 +34,8 @@ def parse_args(args):
                         help="Pandas function/s to summarize columns e.g. mean")
     parser.add_argument("-s", "--sep", default='\t',
                         help="Column separator (default: <TAB>)")
+    parser.add_argument("-n", "--no_sep", action='store_true',
```

Comment:
Can we choose a different option name? This looks like no separator which is defined right above. However it has nothing to do with that and it’s confusing. Also, do we generally use underscores in option names ? I thought we prefer dashes